src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,14 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
-        return new Response($this->serializer->serialize($exception, $format['key']), $exception->getStatusCode(), $headers);
+        $response = new Response(null, $exception->getStatusCode(), $headers);
+
+        if (!$this->debug && $response->isServerError()) {
+            $exception->setMessage(Response::$statusTexts[$response->getStatusCode()]);
+        }
+
+        $response->setContent($this->serializer->serialize($exception, $format['key']));
+
+        return $response;
```

Comment:
@soyuka , Yes in fact, I need the Response object to use the method isServerError() in the Error condition just above.
It was a point I have to discuss with you. No other way to deal it